Formula/ace.rb

```diff
@@ -5,6 +5,11 @@ class Ace < Formula
   sha256 "8bfa251186d9ed1ce48d8936bdace4bc11a49f5fa216f8c1d0be7a65bcf66b39"
   license "DOC"
 
+  livecheck do
+    url "https://github.com/DOCGroup/ACE_TAO/releases/latest"
+    regex(%r{href=.*?/tag/.+TAO\-v?(\d+(?:_\d+)+)["' >]}i)
```

Comment:
```suggestion
    regex(%r{href=.*?/tag/ACE(?:%2B[A-Z]+)*?[._-]v?(\d+(?:[._]\d+)+)["' >]}i)
```

This is the `ace` formula, so we should be specifically matching `ACE` releases. The regex here is instead matching tags that contain `TAO`. This only works at the moment because `ACE` tags start with `ACE+TAO` but it may not be true in the future.

`ACE` tags have taken the following formats over time:

* `ACE-1_2_3`
* `ACE+TAO+CIAO-1_2_3`
* `ACE+TAO-1_2_3`.

The suggested regex above matches all of these, so it's probably a safer bet. It would fail to match a tag that doesn't start with `ACE-` or `ACE+` (URL-encoded to `ACE%2B`) but that doesn't seem like a problem we're likely to encounter in the near future.

Reply:
noted 👍